work15/style.scss

```diff
@@ -0,0 +1,112 @@
+body {
+    font-family: sans-serif;
+}
+
+.loading {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+}
+
+.container {
+    width: 100%;
+    height: 100vh;
+}
+
+.list {
+    img {
+        width: 30px;
+        vertical-align: middle;
+    }
+}
+
+.modal {
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0,0,0,0.5);
+    
+    &_wrap {
+        position: absolute;
+        padding: 10px;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%,-50%);
+        width: 50%;
+        height: 250px;
+        box-shadow: 0 5px 8px 0 rgba(0,0,0,0.2),0 7px 20px 0 rgba(0,0,0,0.17);
+        background-color: #f4f4f4;
+        animation-name: modalopen;
+        animation-duration: 1s;
+
+        p {
+            text-align: center;
+            margin-bottom: 0;
+            font-size: 20px;
+            font-weight: bolder;
+        }
+    }
+
+    @keyframes modalopen {
+        from {opacity: 0}
+        to {opacity: 1}
+    }
+
+    &_buttonArea {
+        display: block;
+        margin-top: 10px;
+        margin-left: auto;
+    }
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 250px;
+    margin-top: 30px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+table {
+    margin-right: auto;
+    margin-left: auto;
+
+    input {
+        padding: 10px 15px;
+    }
+}
+
+label {
+    vertical-align: middle;
+}
+
+.buttonArea {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    max-width: 150px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+}
+
+.button {
+    background: lightblue;
+    color: #fff;
+    padding: 10px 15px;
+    border: 0;
+    border-radius: 5px;
+    font-weight: 900;
+
+    &:hover {
+        background: lightcoral;
+        cursor: pointer;
+    }
```

Comment:
IMO
The css is not part of the spec, but I couldn't help but notice it, so I commented.
I think sending is a positive action, but the color red is being used.
If a color close to red is used, one would expect a cancel button, etc.

[Buttons | Bootstrap](https://getbootstrap.jp/docs/5.0/components/buttons/)
I looked at the bootstrap as an example, and red is still not a common send button color.
Why don't you think a little about the following examples?
[How Button Color Contrast Guides Users to Action](https://uxmovement.com/buttons/how-button-color-contrast-guides-users-to-action/)

Reply:
nits
Sorry for the detail.
Pressing the Cancel button gives the user the impression that the action will be executed. It would be better not to color it.
[キャンセルボタンに色をつけてはいけない理由](https://coliss.com/articles/build-websites/operation/design/why-cancel-buttons-should-never-have-a-color.html)

Reply:
I have confirmed the URL of the link.
This is the first I've heard of this...
I had no idea about UI, I need to learn 💦.

commit: [21b0eea](https://github.com/yasu-pro/js-handson/pull/17/commits/21b0eea079c0201a86ea93f0667307e7d8629561)

Reply:
I have modified the css of the button again.

[76813f1](https://github.com/yasu-pro/js-handson/pull/17/commits/76813f1ec8c48b7c7842a029c12510d8d791ca78)

Reply:
Sorry for repeating myself.
This way please.

commit: [c83477f](https://github.com/yasu-pro/js-handson/pull/17/commits/c83477f189f2a7e0bf6b3b206285536fc16d4237)